src/app/tasks/checker-inbox-and-tasks-tabs/loan-approval-tabs/rejected-kyc-verification/rejected-kyc-verification.component.ts

```
@@ -0,0 +1,192 @@
+    let approvedAccounts = 0;
+    this.batchRequests = [];
+    let reqId = 1;
+    listSelectedAccounts.forEach((element: any) => {
+      const url = 'loans/' + element.id + '?command=approve';
+      const bodyData = JSON.stringify(formData);
+      const batchData = { requestId: reqId++, relativeUrl: url, method: 'POST', body: bodyData };
+      this.batchRequests.push(batchData);
+    });
+    this.tasksService.submitBatchData(this.batchRequests).subscribe((response: any) => {
+      response.forEach((responseEle: any) => {
+        if (responseEle.statusCode = '200') {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)